File: aspnet-core/DailyRoutine.Core/Routine/Routine.cs
using DailyRoutine.Core.Authorization.User;
using System;
using System.Collections.Generic;
using System.Text;

namespace DailyRoutine.Core.Routine
{
    public class Routine
    {
        public int Id { get; set; }
        public int? UserId { get; set; }
        public virtual User User { get; set; }
        public string Summary { get; set; }
        public string Description { get; set; }
        public int Hour { get; set; }
        public int Minute { get; set; }
    }
}

File: react-native/models/routine.js
class Routine {
    Id;
    UserId;
    User;
    Summary;
    Description;
    Hour;
    Minute;
    constructor(data) {
      if (data) {
        this.Id = data.id;
        this.UserId = data.userId;
        this.User = data.user;
        this.Summary = data.summary;
        this.Description = data.description;
        this.Hour = data.hour;
        this.Minute = data.minute;
      }
    }
  }
  export default Routine;
  